packages/help-extension/src/index.tsx

```diff
@@ -117,10 +117,14 @@ function activate(
       text: trans.__('Jupyter Reference'),
       url: 'https://jupyter.org/documentation'
     },
+    {
+      text: trans.__('Jupyter Forum'),
+      url: 'https://discourse.jupyter.org/c/jupyterlab'
+    },
     {
       text: trans.__('Markdown Reference'),
       url: 'https://commonmark.org/help/'
-    }
+    },
```

Comment:
Looks like the `lint` check is failing, which might be related to this trailing comma.